openpype/pipeline/create/subset_name.py

```diff
@@ -154,4 +158,10 @@ def get_subset_name(
         for key, value in dynamic_data.items():
             fill_pairs[key] = value
 
-    return template.format(**prepare_template_data(fill_pairs))
+    try:
+        return template.format(**prepare_template_data(fill_pairs))
+    except KeyError as exp:
+        raise KeyError(
```

Comment:
I still think we should not raise `KeyError`, but custom exception.

Reply:
Changed to new `MissingKeyError` exception